cli/lib/build.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+fn main() {
+  let commit_hash = git_commit_hash();
+  println!("cargo:rustc-env=GIT_COMMIT_HASH={}", commit_hash);
+  println!("cargo:rerun-if-env-changed=GIT_COMMIT_HASH");
+  println!(
+    "cargo:rustc-env=GIT_COMMIT_HASH_SHORT={}",
+    &commit_hash[..7]
+  );
+  let ts_version = "5.6.2";
+  println!("cargo:rustc-env=TS_VERSION={}", ts_version);
```

Comment:
Nvm, it's here. It's strange that `TS_VERSION` is emitted by `cli/lib/` - I thought the idea was that TS shouldn't be available for `denort`

Reply:
I'm going to inline this and fix it

Reply:
Also, yeah it's unfortunate, but we need the typescript version in denort still for the Deno.versions.typescript property. That property should have ideally never been exposed, but not a big deal.